media/css/firefox/mobile.scss

```diff
@@ -0,0 +1,986 @@
+    }
+
+    @media #{$mq-phone-wide} {
+        .page-article-heading {
+            @include at2x('/media/img/firefox/mobile/intro-firefox.png', 352px, 318px);
+        }
+    }
+
+    @media #{$mq-desktop} {
+        .features-scroller-section.page-section:before {
+            background-color: #003eaa;
+            background-image: linear-gradient(to bottom, #003EAA 0%, #004EC2 50%, #0080FF 100%);
+        }
+    }
+}
+
+#focus {
+    .page-article-heading {
+        @include at2x('/media/img/firefox/mobile/intro-focus.png', 280px, 253px);
+    }
+
+    @media #{$mq-phone-wide} {
+        .page-article-heading {
+            @include at2x('/media/img/firefox/mobile/intro-focus.png', 352px, 318px);
+        }
+    }
+
+    @media #{$mq-desktop} {
+        .features-scroller-section.page-section:before {
+            background-color: #C42482;
+            background-image: linear-gradient(148deg, #C42482 0%, #961E7D 20%, #8D1D7C 30%, #721979 60%);
+        }
+    }
+}
+
+.page-nav {
+    color: #fff;
+    display: none;
+
+    a {
+        color: #fff;
+        text-decoration: none;
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Features scroller nav customizations
+
+@media #{$mq-desktop-wide} {
+    .features-scroller-section.page-section .features-scroller-header ul:before {
+        height: 33.33%;
+    }
+
+    #firefox .features-scroller-section.page-section .features-scroller-header .features-scroller-nav {
+        &[data-current="firefox-sync"] ul:before {
+            @include transform(translateY(0));
+        }
+
+        &[data-current="firefox-privacy"] ul:before {
+            @include transform(translateY(100%));
+        }
+
+        &[data-current="firefox-extensions"] ul:before {
+            @include transform(translateY(200%));
+        }
+    }
+
+    #focus .features-scroller-section.page-section .features-scroller-header .features-scroller-nav {
+        &[data-current="focus-privacy"] ul:before {
+            @include transform(translateY(0));
+        }
+
+        &[data-current="focus-speed"] ul:before {
+            @include transform(translateY(100%));
+        }
+
+        &[data-current="focus-lightweight"] ul:before {
+            @include transform(translateY(200%));
+        }
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Icon'ed features for each panel in each product
+
+.features {
+    margin: 1.5em 1em;
+
+    li {
+        padding: 20px 0;
+
+        &:before {
+            background-position: center center;
+            background-repeat: no-repeat;
+            content: '';
+            display: block;
+            height: 40px;
+            margin: 0 auto 1em;
+            width: 40px;
+        }
+
+        &:last-child {
+            border-bottom: none;
+        }
+
+        &.feature-sync:before {
+            background-image: url('/media/img/firefox/mobile/icon-sync.svg');
```

Comment:
Not to worry if we're short on time, but could these small svg icons be sprited to cut down requests?

Reply:
I have a note to do this if time permits. For now, we'll go as-is.